src/main/java/meditracker/argument/ArgumentParser.java

```diff
@@ -22,34 +23,70 @@ class ArgumentParser {
      * @param rawInput Raw input to be parsed
      * @throws ArgumentNotFoundException Argument flag specified not found
      */
```

Comment:
Missing JavaDoc for DuplicateArgumentFoundException

Reply:
Resolved in 236b662